ridgeback/settings.py

```diff
@@ -254,3 +254,7 @@
 
 STATIC_ROOT = 'ridgeback_staticfiles'
 STATIC_URL = '/static/'
+
+# App Cache Configuration
+
+APP_CACHE = os.environ.get('RIDGEBACK_APP_CACHE', '/tmp')
```

Comment:
I'm not sure how long files live in `/tmp` in LSF, but typically they're automatically deleted after some time. We should probably change this to something else

Reply:
We can't actually use the /tmp, because it is not shared. I put the shared directory in configuration, `/tmp` is only ok for local testing